Gemfile

```diff
@@ -192,14 +201,22 @@ ENV['DATABASE_ADAPTER'] ||=
     'mysql2'
   end
 
-if_true(ENV['DATABASE_ADAPTER'].strip == 'postgresql') do
-  gem 'pg', '~> 0.18.3'
+if_true(ENV['DATABASE_ADAPTER'].strip == 'postgresql' || !!ENV['INSTALL_ALL_DBS']) do
```

Comment:
That's a good idea! I bet the additional space needed in the image is not much compared to the improved startup time? Maybe rename it to `INSTALL_ALL_DATABASE_ADAPTERS`?